index.html

```diff
@@ -4,7 +4,7 @@
 ---
 
 <h1 class="page-heading">EIPs
-  <a href="https://discord.io/EthCatHerders"><img src="https://dcbadge.vercel.app/api/server/Nz6rtfJ8Cu?style=flat" alt="Discord channel for ECH eip-editer"></a>
+  <a href="https://discord.com/invite/Nz6rtfJ8Cu"><img src="https://dcbadge.vercel.app/api/server/Nz6rtfJ8Cu?style=flat" alt="Discord channel for ECH eip-editer"></a>
```

Comment:
```suggestion
  <a href="https://dsc.gg/ech"><img src="https://dcbadge.vercel.app/api/server/Nz6rtfJ8Cu?style=flat" alt="Discord channel for ECH eip-editer"></a>
```
This is the alternate shortlink.